src/main/groovy/org/labkey/gradle/plugin/MultiGit.groovy

```diff
@@ -183,23 +183,14 @@ class MultiGit implements Plugin<Project>
         private Boolean isPrivate = false
         private Boolean isExternal = false
         private Boolean isArchived = false
-        private Boolean isSvn = false
```

Comment:
But what if we decide to go back to SVN? Git is good, but ... sigh ... nostalgia.

Reply:
I'd advocate for CVS